content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -50,21 +56,12 @@ the appropriate snapshot.  For example, if you want to run with snapshot 6, use
 [URL](https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/1.1.0-snapshot.6/charts) in installation step 1 below.
 {{< /tip >}}
 
-1.  Update Helm's local package cache with the location of the Helm daily release:
+1. Update Helm's local package cache with the location of the Helm daily release:
```

Comment:
Based on what I said below, I would move this and the working directory creation along with the fetch inside Option #1.